src/include/simeng/arch/aarch64/Instruction.hh

```diff
@@ -453,6 +453,8 @@ class Instruction : public simeng::Instruction {
 
   void setMemoryAddresses(std::vector<MemoryAccessTarget>&& addresses);
 
+  void setMemoryAddresses(MemoryAccessTarget address);
```

Comment:
Shouldn't this be named `setMemoryAddress` given that it is adding just one address